### PR TITLE
Update euclide to 0.19

### PR DESCRIPTION
**Pull request by paulrouget**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "plane-split"
-version = "0.10.0"
+version = "0.10.1"
```
Comment (kvark) [inline Cargo.toml]: I'm afraid we can't just patch update here. `euclid` is in public API, and thus this becomes a breaking change

**Comment by paulrouget** [comment]
Bumped to `0.11.0`.

**Comment by kvark** [comment]
https://crates.io/crates/plane-split/0.11.0

**Pull request merged**